data/src/main/kotlin/com/alfresco/content/data/AnalyticsRepository.kt

```diff
@@ -111,11 +110,40 @@ enum class EventName(val value: String) {
     OpenWith("Event_openWith")
 }
 
+/**
+ * Marked as PageView enum
+ */
+enum class PageView(val value: String) {
+    Recent("PageView_Recent"),
```

Comment:
If possible, can we add the prefix "PageView_" in the function only?

Reply:
It's possible but that would be a string type of value that we will pass to function but right now, I am accepting it as predefined enum

Reply:
Use "Recent" as the predefined enum and pass it in the function. Add prefix in the called function only. Thoughts?